localization/english/wc_regiment_l_english.yml

```
@@ -14,7 +14,155 @@
+ savage_flavor:0 "#F The world believes us barbarians and monsters. Unfortunately, our most savage warriors do little to disprove this. Despite their flaws, they are highly capable warriors, nigh-unstoppable in battle."
+ sea_giant_flavor:0 "#F What they lack in loyalty, our algae-covered Sea Giant slaves more than make up for in strength. Few are the foes who would dare challenge them, and fewer still would succeed."
+ sentinel_flavor:0 "#F Being the main force of the Kaldorei, Sentinels adopt flexible strategies in combat, particularly favoring ambushes or hit-and-run tactics."
+ siren_flavor:0 "#F The Naga are descended from the Highborne, and their ancestry is particularly clear with Naga Sirens, whose mastery of magic is undeniable."
+ snow_hunter_flavor:0 "#F Ours is a hard land, fraught with peril. The bravest of our people, the snow hunters, embrace the perils of winter, and are stronger for it."
+ swarm_flavor:0 "#F The remnants of the Aqir are an undeniable threat, though they differ from most races in favoring quantity over quality in their soldiers, apparent through their swarming tactics."
+ thornguard_flavor:0 "#F Quilboar are well known for their ferocity in battle. The elite Thornguard take this to an entirely new level, tearing through opposition like a scythe through wheat."
+ tidehunter_flavor:0 "#F Aaaaaughibbrgubugbugrguburgle! Mrgllll glrrm gl!"
+ ursa_flavor:0 "#F The indomitable Ursas pounce with ferocious fury on all intruders who threaten their domain and nature itself."
+ warbrave_flavor:0 "#F The bravest of the Tauren have passed many tests of body and spirit and are at the forefront of preserving the legacy of their ancestors and the Earth Mother."
+ war_dinosaur_flavor:0 "#F Being a magnificent relic of a forgotten age, none would question the power of a dinosaur. Truly, only our proud species is worthy of such grand steeds."
+ wolfrider_flavor:0 "#F Orcs and wolves share an unbreakable bond. Individually strong, but together, unstoppable. Skilled, cunning, and fast, wolfriders are among the most deadly enemies one could encounter."
+ wyvern_rider_flavor:0 "#F Together with our new allies, the wyverns, we have created a unit of skilled aerial combatants, the Wind Riders. At long last, the skies are within our grasp."
+
+ ##Early Medival
+ baradian_marine_flavor:0 "#F The elite of our military, Baradian Marines know no fear, only glorious victory. In peacetime they guard the prison of Tol Barad, but it is in war that they truly shine."
+ bearpaws_flavor:0 "#F Among the pandaren, few are as physically powerful as the Bearpaws. Clad in bamboo armor and possessing formidable skill in martial arts, Bearpaws are the first line of defense for the Pandaren."
```

```suggestion
 bearpaws_flavor:0 "#F Among the Pandaren, few are as physically powerful as the Bearpaws. Clad in bamboo armor and possessing formidable skill in martial arts, Bearpaws are the first line of defense for the Pandaren."
```
Capitalized Pandaren so it's consistent.